bcda/models/models.go

```diff
@@ -265,6 +270,32 @@ func AssignAlphaBeneficiaries(db *gorm.DB, aco ACO, acoSize string) error {
 	return db.Exec(s).Error
 }
 
+type CCLF8File struct {
```

Comment:
Is there any reason to make this specifically CCLF8 files?  If we used the CCLFMetaData struct defined in CLI.go (with minor changes) we could store CCLF8 and 9 files in here with minimal effort.

Reply:
Good idea. Since this applies to CCLF9 also, @tbellj, any thoughts? Do you want to do that in your PR first?

Reply:
hrmmm now thinking after seeing these new changes that it'll make more sense if the cclf0 validation pr got merged first. This is a lot more changes since the last time I looked at it :)

Reply:
Kept `cclfMetadata` as a separate struct, but made a shared `CCLFFile` model for 8 and 9.